onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorResize.cpp

```diff
@@ -250,6 +250,11 @@ class DmlOperatorResize : public DmlOperator, public ResizeHelper
         std::string mode = kernelCreationContext.GetOptionalAttribute<std::string>(AttrName::Mode, "NEAREST");
         DML_INTERPOLATION_MODE interpolationMode = Dml::MapStringToInteropolationMode(mode);
 
+
+#if DML_TARGET_VERSION >= 0x6300
```

Comment:
We shouldn't forget to remove this once we release everything. Maybe having a general bug to track this would make sure it doesn't get forgotten.

Reply:
Good point. Bug filed to keep track: https://microsoft.visualstudio.com/OS/_sprints/taskboard/WinML/OS/2401?workitem=48488516